src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -73,7 +77,26 @@ private static JsonObject createDeviceStatusRadioObject(
 	public static JsonArray createDeviceStatus(String band, int channel) {
 		JsonArray jsonList = new JsonArray();
 		jsonList.add(createDeviceStatusRadioObject(
-			band, channel, DEFAULT_CHANNEL_WIDTH
+			band, channel, DEFAULT_CHANNEL_WIDTH, DEFAULT_TX_POWER
+		));
+		return jsonList;
+	}
+
+	/**
+	 * Create the device status array.
+	 *
+	 * @param band band (e.g., "2G")
+	 * @param channel channel number
+	 * @param txPower tx power in dBm
```

Comment:
nit: Line 384 uses () to specify the unit. It's better to keep the same style.

Reply:
Different parts of the code do it differently (even outside my code). If you want to clean this up, as we discussed, please do it in a separate PR